base/params/list.go

```diff
@@ -31,6 +86,13 @@ func (lp List) Validate() error {
 	return nil
 }
 
+// IsEmpty determines if the List struct is empty
+func (lp List) IsEmpty() bool {
+	return lp.Limit == 0 && lp.Offset == 0 &&
+		(lp.Filter == nil || len(lp.Filter) == 0) &&
```

Comment:
No need to check for nil. `len` will do that, and return 0 for a nil slice.

Reply:
thank you, I was having nil pointer dereference trauma